### ISSUE TITLE
Skype for Business 2016: Content of Chat Messages Not Automatically Read

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#7511]
When running the latest NVDA Next snapshot as of this issue, I've noticed that the content of chat messages is not automatically read. From taking a look at the latest couple of merges, it looks as if these should now automatically be read. Is this the case?

**Comment by ehollig** [comment]
This is a duplicate of #5333 and should have been fixed in #7281 in NVDA 2017.3. If this is not the case, CC @michaelDCurran

**Comment by elliott94** [comment]
@michaelDCurran - Am I right in thinking that NVDA will only read the initial message that gets sent with this pull?
